addons/point_of_sale/views/account_journal_view.xml

```diff
@@ -1,27 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record id="action_account_journal_form" model="ir.actions.act_window">
```

Comment:
See with @lap-odoo if we want to keep menus linked to journals in POS or if we only want to show payment methods. If we don't want journals, we can delete file.